src/main/java/com/meturial/domain/recipe/service/RecipeService.java

```diff
@@ -29,6 +27,24 @@ public class RecipeService {
     private final RecipeSequenceRepository recipeSequenceRepository;
     private final ReviewRepository reviewRepository;
 
+    @Transactional(readOnly = true)
+    public QueryCategoryResponse queryCategory() {
+        List<Category> categoryList = recipeRepository.queryCategory();
+        List<CategoryElement> categoryElements = categoryList
+                .stream()
+                .map(this::buildCategoryElement)
+                .toList();
+
```

Comment:
그냥 List 가져온 후에 바로 stream 사용하면 좋을 듯 합니다
```suggestion
    public QueryCategoryResponse queryCategory() {
        List<CategoryElement> categoryList = recipeRepository.queryCategory()
                 .stream()
                .map(this::buildCategoryElement)
                .toList();

```

Reply:
요런식으로여

Reply:
넵!